sim/dex-data.ts

```diff
@@ -606,7 +606,9 @@ export class Template extends BasicEffect implements Readonly<BasicEffect & Temp
 	 */
 	readonly learnset?: {[moveid: string]: MoveSource[]};
 	/** Source of learnsets for Pokemon that lack their own */
-	readonly inheritsFrom: string | string[];
+	readonly inheritsFrom?: string | string[];
+	/** Whether or not a Pokemon should be hidden in the teambuilder */
+	readonly hide?: 'SpriteOnly' | 'SpriteMinute' | 'Minute';
```

Comment:
Neither the word `hide` nor these possible values are very clear to me.

`SpriteOnly` appears to be a cosmetic form? I don't understand `SpriteMinute` and `Minute` at all.

Reply:
`SpriteOnly` refers to formes that only differ from their base in appearance. The difference with cosmetic formes is that their appearance can convey competitively-relevant information. For example if you see Mimikyu-Busted you know that Disguise is no longer in play.

`Minute` (as in, the adjective meaning small/insignificant, and not the unit of time) refers to formes that do technically have distinctions from their base, but not very many. It makes sense to tier these mons together. Any strategy we write would cover both of them. We'd display them on the same Dex page, noting the few differences. Basculin-Blue-Striped is a good example.

Per Discord discussion I'm not sure what `SpriteMinute` is supposed to be.

Reply:
I don't like `Minute`, maybe something more like `Minor`.

I also don't like `hide`, I think I'd prefer `unimportantForme`.

Reply:
I also want the "can convey competitively-relevant information" clarification in the doc comment.